be/src/olap/rowid_conversion.h

```diff
@@ -115,6 +121,7 @@ class RowIdConversion {
     // value indicates row id of destination segment.
     // <UINT32_MAX, UINT32_MAX> indicates current row not exist.
     std::vector<std::vector<std::pair<uint32_t, uint32_t>>> _segments_rowid_map;
```

Comment:
Why not replace to MyVector here (just like what you did in SimpleRowidConversion)?

Reply:
because _segments_rowid_map is used by clucene, modify _segments_rowid_map's template means clucene's code should alse be modifed, it's too complex.